cmd/server/main.go

```diff
@@ -71,6 +86,10 @@ func createAndroidVariantConfigMap(variant *androidVariant) {
 				"mobile":       "enabled",
 				"serviceName":  "ups",
 				"resourceType": "binding",
+
+				// Used by the mobile-cli to discover config objects
+				"serviceInstanceId": pushClient.serviceInstanceId,
+				"mobileClientID": clientId,
```

Comment:
ping @maleck13 Every variant config maps will have those fields. Instead of organizing multiple variants in an array, can we use their type as the key in an object like here: https://github.com/aerogear/aerogear-cordova-push/blob/master/example/push-config.json ?

Also not sure what's the best way to add the `pushServerURL` from the example above.

Reply:
that would be actually a bit nicer